db/migrate/20161214221229_reencrypt_email_with_salt.rb

```diff
@@ -1,29 +1,33 @@
 class ReencryptEmailWithSalt < ActiveRecord::Migration
   def up
-    rotate(Figaro.env.password_pepper, Figaro.env.email_encryption_key)
+    new_salt = Figaro.env.attribute_encryption_key
+    rotate(Figaro.env.password_pepper, new_salt)
   end
 
   def down
-    rotate(Figaro.env.email_encryption_key, Figaro.env.password_pepper)
+    old_salt = Figaro.env.attribute_encryption_key
+    rotate(old_salt, Figaro.env.password_pepper)
+  end
+
+  class EncryptedEmail < EncryptedAttribute
+    def self.new_from_email(plain_email, old_uak)
+      new_from_decrypted(plain_email, old_uak)
+    end
```

Comment:
Also, do we even need to define this here? Can we not call `EncryptedAttribute#new_from_decrypted` directly?